### PR TITLE
added recipe content, added bootstap link in our story head

### PR DESCRIPTION
**Pull request by cwlainson**
added recipe content added bs link to head.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,6 +1,10 @@
 <!DOCTYPE html>
 <html>
 
+<head>
+	<!DOCTYPE html>
```
Comment (billythakidd04) [inline index.html]: this doesn't belong here

File: ourstory.html [hunk 2]
```diff
@@ -2,7 +2,8 @@
 <html>
 
 <head>
-    <title>PizzaParty</title>
+    <title>PizzaParty</title><link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta2/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-BmbxuPwQa2lc/FVzBcNJ7UAyJxM6wuqIj61tLrc4wSX0szH/Ev+nYRRuWlolflfl" crossorigin="anonymous">
```
Comment (billythakidd04) [inline ourstory.html]: need to format this file. the link tag should be on its own line

File: ourstory.html [hunk 3]
```diff
@@ -15,7 +16,7 @@ <h1 style="background-color:green;">Our Story</h1>
         Buffalo in 1950. The brothers opened up <em>"Pizzeria di Pardi"</em> on Grant street in Buffalo's<br/>
         west side. In 1986 we expanded to 8 locations in western NY and Pizzeria di Pardi became<br/>
         <b>Pizza Party!</b> Today we have three locations serving up the classic pizza & wings Buffalo <br/>
-        is known for our original location as well as our Hertel Aveneue and UB locations.
+        is known for at our original location as well as our Hertel Aveneue and UB locations.
     </p>
     <img src="/Users/christopherlainson/Desktop/workspace/PizzaParty/img/party_boys.jpg" Alt="Pardi Bros Havana NYE 1958 colorized" style="width:560px;height:325px"><!-- move these styles to css -->
```
Comment (billythakidd04) [inline ourstory.html]: this path is wrong. It only refers to your local machine it needs to be the [relative path](https://desktop.arcgis.com/en/arcmap/10.3/tools/supplement/pathnames-explained-absolute-relative-unc-and-url.htm#GUID-5118AC85-57E4-4027-AC24-FB6E99FADEFF)

File: recipes.html [hunk 4]
```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>PizzaParty</title>
+     <link href="css/style.css" rel="stylesheet" />
```
Comment (billythakidd04) [inline recipes.html]: missing the bootstrap stuff here

File: recipes.html [hunk 5]
```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
```
Comment (billythakidd04) [inline recipes.html]: this file needs formatting as well. Make sure your using 1 tab set to 4 spaces

**Review by billythakidd04 — CHANGES_REQUESTED** [review]
Needs a good formatting overall. Still a few bugs.

**Comment by billythakidd04** [comment]
@cwlainson is there an update on this?!

**Comment by billythakidd04** [comment]
BUMP

**Comment by cwlainson** [comment]
I made changes and pushed them but it’s not showing up on GitHub

**Comment by cwlainson** [comment]
Actually the formatting was [really bad] still I think I’ll change it and try
again

File: index.html (hunk 1 above)
Comment (cwlainson) [reply index.html]: removed extra doctype & html

File: ourstory.html (hunk 2 above)
Comment (cwlainson) [reply ourstory.html]: tag put on its own line
formatting done

File: recipes.html (hunk 4 above)
Comment (cwlainson) [reply recipes.html]: bootstrap added

File: ourstory.html (hunk 3 above)
Comment (cwlainson) [reply ourstory.html]: image path corrected

File: recipes.html [hunk 6]
```diff
@@ -1,112 +1,130 @@
...
+			<li>Fresh Mozzarella Cheese</li>
+			<li>Cubed Provolone</li>
+			<h3>Garnish & Dressing</h3>
+			<li>Garnish with 2 sliced, hard boiled eggs, 1 pepperonchini per person
+					and ground Black Pepper to taste.</li>
+			<p>
+				Regular Italian dressing will do or whatever dressing you prefer...just
+				no Ranch, but we think our homemade Vinaigrette is perfect for this salad.
+			</p>
+				</ul>>
```
Comment (rbaia28) [inline recipes.html]: Would you be able to remove the second > on line 121?

Comment (rbaia28) [reply recipes.html]: connected with chris and this is handled

**Review by rbaia28 — DISMISSED** [review]
Looks good to me!

File: ourstory.html [hunk 7]
```diff
@@ -42,25 +45,31 @@
...
+	<p>Royal Treatment</p>
+	<iframe width="560" height="315" src="https://www.youtube.com/embed/CPEwAA6G42I?start=15" frameborder="0"
+		allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture"
+		allowfullscreen></iframe>
 
-    <p>Royal Treatment</p>
-    <iframe width="560" height="315" src="https://www.youtube.com/embed/CPEwAA6G42I?start=15" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
-    
-    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta2/dist/js/bootstrap.bundle.min.js" integrity="sha384-b5kHyXgcpbZJO/tY9Ul7kGkf1S0CWuKcCD38l8YkeH8z8QjE0GmW1gYU5S9FOnJ0"crossorigin="anonymous"></script>
+	<script src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta2/dist/js/bootstrap.bundle.min.js"
```
Comment (billythakidd04) [inline ourstory.html:70]: this should be the **last** thing before `</body>`

**Review by billythakidd04 — CHANGES_REQUESTED** [review]
close all `<br/>` and `<hr/>` tags and fix the position of the bootstrap js on the ourstory file and you're good.

**Review by billythakidd04 — APPROVED** [review]

**Pull request merged**